### ISSUE TITLE
unsupported locale setting exception while doing get_new_log 

### ISSUE THREAD

**Issue by TomasTomecek** [rebase-helper/rebase-helper#867]
I'm a little confused by the stack trace to be honest

https://sentry.io/share/issue/22614a85f2564b52afb428c931345706/

locale is set to 'en_US.UTF-8' while old_locale, in the frame above, is a list:
```
[
'en_US', 
'UTF-8'
]
```

this is happening in packit production in a standard fedora:33 container, namely `quay.io/packit/base`

**Comment by TomasTomecek** [comment]
noiw I understand why it's not working for us; since it's a "minimal" container image, it does not contain `en_US.UTF-8` locale even though we set it in our dockerfile, hence the subsequent `setlocale` call fails

we only have these:
```
[root@a512958eef7e /]# locale -a
C
C.utf8
POSIX
```

**Comment by nforro** [comment]
I guess we could fallback to setting C locale, or just silently ignore the error.

**Comment by TomasTomecek** [comment]
> I guess we could fallback to setting C locale, or just silently ignore the error.

falling back sounds like a good strategy to me; anyway, I fixed it for us ^

**Comment by nforro** [comment]
Closed via #868.
